Content.Shared/WebPlacer/WebPlacerComponent.cs

```diff
@@ -0,0 +1,81 @@
+using Content.Shared.Actions;
+using Content.Shared.Whitelist;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.WebPlacer;
+
+/// <summary>
+/// Gives the entity (probably a spider) an ability to spawn entites (probably webs) around itself.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+[Access(typeof(SharedWebPlacerSystem))]
+public sealed partial class WebPlacerComponent : Component
+{
+    /// <summary>
+    /// Id of the entity getting spawned.
+    /// </summary>
+    [DataField]
+    public EntProtoId WebPrototype = "SpiderWeb";
+
+    /// <summary>
+    /// Id of the action that will be given.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    [DataField]
+    public EntProtoId SpawnWebAction = "ActionSpiderWeb";
+
+    /// <summary>
+    /// Action given to the player.
+    /// </summary>
+    //[ViewVariables]
```

Comment:
why its commented? Where DataField

Reply:
I didn't add DataField because this variable is getting set on init, and to my understanding EntityUid can't be set in .yml so it felt unnecessary.
VV being commented was just a mistake on my part.